recipes/r-motifbinner/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set version = "2.0.0" %}
+{% set name = "MotifBinner" %}
+
+package:
+  name: r-{{ name | lower }}
+
+  version: {{ version }}
+
+source:
+  url:
+    - https://github.com/HIVDiversity/{{ name }}2/archive/v{{ version }}.tar.gz
+  sha256: 6337bf7a7c84c4d3dbdf784b1acc6ad92f05b97fe4068b58ef19630da0c5590d
+
+build:
+  noarch: True
+  number: 0
+  script: "$R CMD INSTALL --build ."
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  host:
+    - r-base
+    - r-stringdist
+    - r-dplyr
+    - r-domc
+    - r-ggplot2
+    - r-knitr
+    - r-optparse
+    - r-rmarkdown
+    - r-foreach
+    - r-tidyr
+    - r-gridextra
+    - r-reshape2
+    - r-stringi
```

Comment:
These deps here do not match the one in the run section. Is really everything needed? And what about r-stringi which is only listed here?